internal/backend/runtime/omni/controllers/omni/etcd_backup_test.go

```diff
@@ -82,6 +80,8 @@ func (suite *EtcdBackupControllerSuite) register2(ctrl controller.Controller, er
 }
 
 func (suite *EtcdBackupControllerSuite) SetupTest() {
+	suite.OmniSuite.ctx, suite.OmniSuite.ctxCancel = context.WithCancel(context.Background()) //nolint:fatcontext
```

Comment:
I would put some comment here and in the config patch cleanup to explain that we need a timeout-less context in the synctest-based tests